### PR TITLE
Update GitHub templates

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Update GitHub templates.
Feel free add commits if needed.

## PR Context  

Now we have many issues with bug label what doesn't look right :-( I think the label should be assigned only for confirmed bugs.
Also [DOCS] and others in title look superfluous.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [x] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 name: Bug report 🐛
 about: Report errors or unexpected behavior 🤔
-title: "[BUG] My bug report"
-labels: Issue-Bug
+title: "My bug report"
+labels: Issue-Discussion
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: maybe question would be a better place to start

**Comment by TravisEz13** [comment]
@PaulHigin 	@adityapatwardhan 
Linux failure is read but unrelated to this PR

FPM failed to install

```
2019-01-30T05:25:33.5365431Z ERROR:  Error installing fpm:
2019-01-30T05:25:33.5366069Z 	ERROR: Failed to build gem native extension.
2019-01-30T05:25:33.5366505Z 
2019-01-30T05:25:33.5367277Z     current directory: /var/lib/gems/2.3.0/gems/childprocess-1.0.0/ext
2019-01-30T05:25:33.5367474Z /usr/bin/ruby2.3 mkrf_conf.rb
2019-01-30T05:25:33.5367645Z 
2019-01-30T05:25:33.5368132Z current directory: /var/lib/gems/2.3.0/gems/childprocess-1.0.0/ext
2019-01-30T05:25:33.5368905Z /usr/bin/ruby2.3 -rubygems /usr/share/rubygems-integration/all/gems/rake-10.5.0/bin/rake RUBYARCHDIR=/var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/childprocess-1.0.0 RUBYLIBDIR=/var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/childprocess-1.0.0
2019-01-30T05:25:33.5369492Z /usr/bin/ruby2.3: No such file or directory -- /usr/share/rubygems-integration/all/gems/rake-10.5.0/bin/rake (LoadError)
2019-01-30T05:25:33.5369656Z 
2019-01-30T05:25:33.5369833Z rake failed, exit code 1
2019-01-30T05:25:33.5370252Z 
2019-01-30T05:25:33.5370724Z Gem files will remain installed in /var/lib/gems/2.3.0/gems/childprocess-1.0.0 for inspection.
2019-01-30T05:25:33.5371210Z Results logged to /var/lib/gems/2.3.0/extensions/x86_64-linux/2.3.0/childprocess-1.0.0/gem_make.out
```

```
2019-01-30T05:41:12.6286957Z WARNING: Package dependency 'fpm' not found. Run Start-PSBootstrap -Package
2019-01-30T05:41:12.6746525Z Get-ChildItem : Cannot find path '/usr/lib64/ruby/gems' because it does not exist.
2019-01-30T05:41:12.6746697Z At /home/vsts/work/1/s/tools/packaging/packaging.psm1:1155 char:25
2019-01-30T05:41:12.6746994Z + ...          $gemsPath = Get-ChildItem -Path /usr/lib64/ruby/gems | Sort- ...
2019-01-30T05:41:12.6747073Z +                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2019-01-30T05:41:12.6747366Z + CategoryInfo          : ObjectNotFound: (/usr/lib64/ruby/gems:String) [Get-ChildItem], ItemNotFoundException
2019-01-30T05:41:12.6747431Z + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
2019-01-30T05:41:12.6747528Z  
2019-01-30T05:41:12.6996141Z Dependency precheck failed!
2019-01-30T05:41:12.6997120Z At /home/vsts/work/1/s/tools/packaging/packaging.psm1:1168 char:13
2019-01-30T05:41:12.6997384Z +             throw "Dependency precheck failed!"
2019-01-30T05:41:12.6997624Z +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2019-01-30T05:41:12.6997856Z + CategoryInfo          : OperationStopped: (Dependency precheck failed!:String) [], RuntimeException
2019-01-30T05:41:12.6998078Z + FullyQualifiedErrorId : Dependency precheck failed!
```

File: .github/ISSUE_TEMPLATE/Bug_Report.md (hunk 1 above)
Comment (iSazonov) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]: Sorry, I don't understand the comment.

Comment (adityapatwardhan) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]: I guess @TravisEz13 means the label `Issue-Question`

**Comment by TravisEz13** [comment]
Filed https://github.com/PowerShell/PowerShell/issues/8801 for windows failure and restarting

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**